security-manager/src/main/java/org/wildfly/extension/security/manager/SecurityManagerSubsystemAdd.java

```diff
@@ -62,14 +63,25 @@ protected void performBoottime(final OperationContext context, final ModelNode o
         final ModelNode node = Resource.Tools.readModel(resource);
         // get the minimum set of deployment permissions.
         final ModelNode deploymentPermissionsModel = node.get(DEPLOYMENT_PERMISSIONS_PATH.getKeyValuePair());
-        final ModelNode minimumPermissionsNode = MINIMUM_PERMISSIONS.resolveModelAttribute(context, deploymentPermissionsModel);
-        final List<PermissionFactory> minimumSet = this.retrievePermissionSet(DeferredPermissionFactory.Type.MINIMUM_SET, context, minimumPermissionsNode);
-
-        // get the maximum set of deployment permissions.
-        ModelNode maximumPermissionsNode = MAXIMUM_PERMISSIONS.resolveModelAttribute(context, deploymentPermissionsModel);
+        ModelNode minimumPermissionsNode = null;
+        ModelNode maximumPermissionsNode = null;
```

Comment:
optional, null assignements can be removed, those variables will never be used without being initialized, so can be just:

```
        ModelNode minimumPermissionsNode;
        ModelNode maximumPermissionsNode;
```